lib/ansible/modules/system/user.py

```diff
@@ -726,6 +736,33 @@ def chown_homedir(self, uid, gid, path):
             e = get_exception()
             self.module.exit_json(failed=True, msg="%s" % e)
 
+    def parse_defaults(self):
+        try:
+            f = open('/etc/login.defs', 'r')
+            for line in f:
+                if 'UMASK' in line and '#' not in line.split()[0]:
+                    maskvalue=line.split()[1]
+                    self.newumask = int(maskvalue, 8)
+            f.close()
+        except:
+            pass
+        try:
+            if self.skeleton is None:
+                f = open('/etc/default/useradd', 'r')
+                if 'SKELE' in line and '#' not in line.split()[0]:
```

Comment:
Same comment regarding parsing as the other file ('SKELE' would match 'SKELETON', '#' might be in other place, and we do not verify that '=' is present before splitting. And the wide 'try'/'except' clause should IMHO be changed.